actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -410,6 +410,10 @@ def [](key)
       convert_hashes_to_parameters(key, @parameters[key])
     end
 
+    def to_param
```

Comment:
delegating it to `to_h` is also an option (forgot to mention it in the PR description).